istio.VERSION

```diff
@@ -1,13 +1,13 @@
 # DO NOT EDIT THIS FILE MANUALLY instead use
 # install/updateVersion.sh (see install/README.md)
-export CA_HUB="gcr.io/istio-testing"
-export CA_TAG="replace-by-actual-tag"
-export MIXER_HUB="gcr.io/istio-testing"
-export MIXER_TAG="replace-by-actual-tag"
-export PILOT_HUB="gcr.io/istio-testing"
-export PILOT_TAG="replace-by-actual-tag"
+export CA_HUB="costinm"
```

Comment:
please look at your PR/diff before and after submission :-)

Reply:
Ah, istio.VERSION again...